addons/im_livechat/static/src/views/discuss_channel_list/discuss_channel_list_view_controller.js

```diff
@@ -9,12 +9,13 @@ export class DiscussChannelListController extends ListController {
     }
 
     async openRecord(record) {
-        if (!record.data.is_member) {
-            return super.openRecord(record);
-        }
-        this.actionService.doAction("mail.action_discuss", {
-            name: _t("Discuss"),
-            additionalContext: { active_id: record.resId },
-        });
+        this.actionService.doAction(
+            {
+                name: _t("Discuss"),
+                tag: "mail.action_discuss",
+                type: "ir.actions.client",
+            },
+            { additionalContext: { active_id: record.resId } }
+        );
```

Comment:
how comes the signature of doAction changed in this PR?

Reply:
I hit my head against the wall for an hour because of this, it seems impossible to display ControlPanel component correcty with the previous version of doAction. Unfortunately, I was not able to identify the exact cause.